### PR TITLE
Handle new identifier structure

### PR DESCRIPTION
**Pull request by BerniVarga**

### CODE REVIEW

**Comment by BerniVarga** [comment]
The checks will pass as soon as the rw's get merged. (Soon:)

File: cypher_test.go [hunk 1]
```diff
@@ -156,6 +159,7 @@ func writeSubjects(assert *assert.Assertions, db *neoism.Database, batchRunner *
 }
 
 func deleteSubjects(subjectsRW baseftrwapp.Service) {
+	fmt.Printf("halihó - próbálkozom")
```
Comment (nwrigh) [inline cypher_test.go]: Debug message I presume

**Comment by nwrigh** [comment]
:+1: I took the liberty of taking out a log message from the tests though

**Comment by nwrigh** [comment]
:+1: And I took the liberty of removing a log/debug message

File: cypher_test.go (hunk 1 above)
Comment (BerniVarga) [inline cypher_test.go]: Oh, yes. thanks for removing it :D

**Comment by nwrigh** [comment]
:+1:

**Pull request merged**